src/machine/board_kb2040.go

```diff
@@ -0,0 +1,111 @@
+//go:build kb2040
+// +build kb2040
+
+package machine
+
+import (
+	"device/rp"
+	"runtime/interrupt"
+)
+
+// Onboard crystal oscillator frequency, in MHz.
+const xoscFreq = 12 // MHz
+
+// GPIO Pins
+const (
+	D0  = GPIO0
+	D1  = GPIO1
+	D2  = GPIO2
+	D3  = GPIO3
+	D4  = GPIO4
+	D5  = GPIO5
+	D6  = GPIO6
+	D7  = GPIO7
+	D8  = GPIO8
+	D9  = GPIO9
+	D10 = GPIO10
+)
+
+// Analog pins
+const (
+	A0 = GPIO26
+	A1 = GPIO27
+	A2 = GPIO28
+	A3 = GPIO29
+)
+
+const notConnected = 31
+
+const LED = notConnected
```

Comment:
Please don't include constants for pins that don't exist. If the board has no LED, it should not define `machine.LED`.

Reply:
Ok, that makes sense. For the LED it is no problem I think ... I also had the SPI1 pins set up that way because the RP2040 `machine` implementation expected it to be there and I was mostly transcribing this information from the Arduino core for this board